r-pkg/_pkgdown.yml

```diff
@@ -1,17 +1,25 @@
 template:
+  bootstrap: 5
   params:
     bootswatch: flatly
 
+# this needs to be specified and to match 'URL:' in DESCRIPTION,
+# or pkgdown raises this error:
+#
+#   URLs not ok.
+#   In _pkgdown.yml, url is missing.
+#
+url: https://github.com/uptake/uptasticsearch
+
 repo:
   url:
     home: https://github.com/uptake/uptasticsearch/
     source: https://github.com/uptake/uptasticsearch/tree/main/r-pkg/
     issue: https://github.com/uptake/uptasticsearch/issues
     user: https://github.com/
 
-
 reference:
-  - title: Main function
+  - title: Main function (DELETE ME)
```

Comment:
```suggestion
  - title: Main function
```

TODO: remove this before merging.

Put this here to prove that docs deployed this way are making it to the site: https://uptake.github.io/uptasticsearch/reference/index.html

<img width="567" alt="image" src="https://github.com/user-attachments/assets/23d869f4-d6d8-4a06-9ba1-511f6779ef7a" />